### ISSUE TITLE
allow way to import sentences and assign them to two different owners by lang

### ISSUE THREAD

**Issue by alanfgh** [Tatoeba/tatoeba2#106]
CK writes:

It would be nice in the IMPORT SENTENCES function also had the following 3rd way to import sentences.

Import 2 languages, assigning different usernames as the owners of sentences in each language.

For example, if I created a set of English sentences and had my friend translate them all into Japanese, I could import the whole file at one time, assigning ownership of the English to me and the ownership of the Japanese to my friend.

**Comment by ckjpn** [comment]
I would still like to be able to do this.

Here is some data I currently have to import.

English by User 1396
Berber by User 17985

English + TAB + Berber

That was a life-changing moment. 	Tin d taswiɛt yettbeddilen tudert. 
Tom read Mary's suicide note.	Tom yeɣra tabṛat n tenɣimant n Mary. 
Tom isn't breathing at all. 	Tom ur la yettneffis akk. 
Tom isn't moving at all. 	Tom ur la yettewliwil akk. 
It all happened too fast. 	Kullec yeḍra s tɣawla. 
I'm going to kill myself. 	Ad nɣeɣ iman-inu. 
We know Tom didn't take his own life. 	Neẓra dakken Tom ur yenɣi iman-nnes. 
Do I need to worry about this?	Terra tmara ad agadeɣ seg waya? 
Could you do this? 	Tzemred ad tged aya? 
Nothing worked. 	Acemma ur yečči. 
Tom doesn't believe Mary's story. 	Tom ur yumin taḥkayt n Mary.

**Comment by ckjpn** [comment]
I would still like to be able to do this.

Now that duplicates filtered out as they are imported and proper links of the second language of the pair are automatically linked to the existing sentences in the database, this function would be very helpful.

If such a function were possible, it would be possible to do the following.

1. I could give a list of good English sentences not yet translated into another member's language for him to translate offline.  Then, import them and have them all properly linked to the English sentences.  This would be especially useful for those with slower internet connections, or who are often not connected to the internet.  It would also help lower cost for those living in areas where they pay for internet based on usage and not on a flat fee. (I'm pretty sure there are still places in the world like this.)    Of course, this would not just be limited to English to another language translations.

2. I could accept sentences pairs from members who ask me to proofread the English, then import the English under my username and the other language under the other member's username.  

Sharptoothed and I worked through a public domain English-Russian dictionary this way a few years ago.  I sent him the raw data, he deleted pairs that had bad Russian or rewrote the Russian into modern or correct Russian.  He then sent me the data, and I deleted any pair that had bad English (he agreed to this), or I re-wrote the English and sent those pairs back to him to verify that they still matched in meaning.  After that, I imported the pairs and assigned sentences in both languages to his username.

It would have been helpful at that time if I could have assigned the English to my username instead of his.
